### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by amyblais**
For https://github.com/mattermost/mattermost-server/pull/14565#discussion_r464486344

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -197,6 +197,8 @@ Multiple setting options were added to `config.json`. Below is a list of the add
      - Added ``EnableSearching`` to enable search queries to use bleve search.
      - Added ``EnableAutocomplete`` to enable autocomplete queries to use bleve search.
      - Added ``BulkIndexingTimeWindowSeconds`` to determine the maximum time window for a batch of posts being indexed by the Bulk Indexer.
+ - Under ``Notifications`` under ``Push Notification Contents`` in ``config.json``:
+     - Added ``full`` to enable push notifications with full message content sent in the notification payload.
```
Comment (esethna) [inline source/administration/changelog.md:201]: It's not added, this is just the new default for new installs. Same with link previews is just on by default for new installs now

File: source/administration/changelog.md [hunk 2]
```diff
@@ -197,6 +197,8 @@ Multiple setting options were added to `config.json`. Below is a list of the add
      - Added ``EnableSearching`` to enable search queries to use bleve search.
      - Added ``EnableAutocomplete`` to enable autocomplete queries to use bleve search.
      - Added ``BulkIndexingTimeWindowSeconds`` to determine the maximum time window for a batch of posts being indexed by the Bulk Indexer.
+ - Under ``EmailSettings`` in ``config.json``:
+     - Changed ``PushNotificationServer`` to default ``full`` for new installs.
```
Comment (esethna) [inline source/administration/changelog.md:201]: It's this setting: https://docs.mattermost.com/administration/config-settings.html#push-notification-contents

**Comment by esethna** [comment]
Also this default was changed to true: https://docs.mattermost.com/administration/config-settings.html#enable-link-previews

**Comment by amyblais** [comment]
``enable-link-previews`` is already listed there.

**Pull request merged**